files/bootstrap.sh

```diff
@@ -342,3 +342,31 @@ fi
 systemctl daemon-reload
 systemctl enable kubelet
 systemctl start kubelet
+
+# gpu boost clock
+if  command -v nvidia-smi &>/dev/null ; then
+   echo "nvidia-smi found"
+
+   sudo nvidia-smi -pm 1 # set persistence mode
```

Comment:
Can you add comments on each of these options explaining what they do and why you're setting them to the values you are?

Reply:
- nvidia-smi -pm 1 : this sets persistence mode as enabled, which is a necessary change to be able to change application clock as we want. more info from https://developer.nvidia.com/blog/increase-performance-gpu-boost-k80-autoboost/
- nvidia-smi --auto-boost-default=0 : we disable auto boost mode in favor of application clock. more info from https://developer.download.nvidia.com/compute/DCGM/docs/nvidia-smi-367.38.pdf